model/github_runner.rb

```diff
@@ -44,4 +44,8 @@ def check_pulse(session:, previous_pulse:)
   def self.redacted_columns
     super + [:workflow_job]
   end
+
+  def used_vm_cores
+    vm&.cores.to_i
```

Comment:
Isn't it already an integer? Do we need to use `to_i` to cast it?

Reply:
in case vm is `nil`, I want to return 0. `to_i` handles that for me